### DISCUSSION TITLE
Issues trying to get a homebrew app to launch  " Cherry Tree "

### DISCUSSION THREAD

**Discussion by dmorel89** [Homebrew/discussions#3550] [Everyday usage] [answered]
Good day. I'm not proficient yet in the language of powerShell / Command line. I want to get "CherryTree" the note taking app. I found a way to get it following the instructions

https://formulae.brew.sh/formula/cherrytree

But I can't seem to find it installed . or when I go to the Applications folder. 
I know like in other package managers like in linux some of these libraries need to have a  a line of command executed before using it. Some get installed right away.

I did find this
https://www.phillips321.co.uk/2017/12/06/cherrytree-on-macos-osx/ but I don't know what the other program is  that it needs to run the cherry tree app. 

Can someone help me or if there is already a thread started kindly point me in the right direction. Thank you :)

**Comment by SMillerDev** [comment, marked as answer]
There is no app, there is only a command line invocation. You can type `cherrytree` ik your terminal to interact with it after `brew install cherrytree`

**Reply by dmorel89** [reply]
Thank you so very much for this. Now I can go back and taking notes on my mac. Have a good day.
